Dockerfile

```diff
@@ -1,7 +1,7 @@
 ################################
 # STEP 1 build executable binary
 ################################
-FROM registry.access.redhat.com/ubi8/go-toolset:latest AS builder
+FROM registry.access.redhat.com/ubi9/go-toolset:latest AS builder
```

Comment:
Shouldn't line 38 be changed from `ubi8` to `ubi9` as well?

Reply:
Not sure how that didn't get added from my diff, thanks for spotting it!